cogito/protocol.go

```diff
@@ -247,12 +247,12 @@ func (src *Source) Validate() error {
 	sinks := sets.From(src.Sinks...)
 	if sinks.Size() == 0 || sinks.Contains("github") {
 		// Cogito commit Github status mandatory fields.
```

Comment:
This comment refers to the whole then block, while comment at line 254 refers only to the lines nearby. This is confusing because the two comments are laid out in the same way (no blank lines or similar).

I would put a blank line after line 249 and even remove the comment at line 254 (since the error message is saying exactly the same).